Makefile

```diff
@@ -7,7 +7,7 @@ LEDGER_ENABLED ?= true
 BINDIR ?= $(GOPATH)/bin
 BUILDDIR ?= $(CURDIR)/build
 SIMAPP = ./app
-HTTPS_GIT := https://github.com/terra-money/core.git
+HTTPS_GIT := https://github.com/terra-money/classic-core.git
```

Comment:
I thought this will be: "classic-terra/core" ?

Reply:
It's the TFL repo.

Reply:
It is intended since our repo has no valid main